rb/lib/selenium/webdriver/firefox/profiles_ini.rb

```diff
@@ -57,11 +57,10 @@ def parse
               is_relative = Regexp.last_match(1).strip == '1'
             when /^Path=(.+)$/
               path = Regexp.last_match(1).strip
+              p = path_for(name, is_relative, path)
+              @profile_paths[name] = p if p
```

Comment:
I'm confused by the question... They are being used? I'm not saying this parser is the best way to do it, I just moved 2 lines so that it would work correctly. We can re-write it if you think that's better